blog/2023-08-23-nfdi-grant.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Martin Magdinier
+title: "OpenRefine receives a new NFDI grant to improve its reconciliation features"
+---
+[NFDI](https://www.nfdi.de/?lang=en) has awarded OpenRefine a EUR 10,000 grant to enhance its reconciliation feature. This grant builds upon the progress made in 2022 with Wikibase and Wikimedia Commons integration and paves the way for more user-friendly reconciliation services, connecting with the [50+ existing reconciliation services](https://reconciliation-api.github.io/testbench/#/).
```

Comment:
The idea was to reference work done in 2022 when  creating the Wiki Commons integration.

Reply:
@wetneb  let me know if the version in the last commit works for you. If it does, feel free to merge the PR. 
thank you

Reply:
I think it is still curious in the current form, because the wiki commons project did not make any improvements to reconciliation as far as I can remember. So I would not phrase it as "continuation of progress" from that grant. If you want to reference this grant, I think a better way to phrase this would be to say that the NFDI grant will address issues that have been flagged during the user testing phase of the Wiki Commons grant.

Reply:
I think the design work that was paid by the Wiki Commons grant should technically also count as progress, no? Because we did some extensive testing and then wireframing of how results are rendered, how the enrichment process can be improved + things like image reconciliation, addition of thumbnails, etc. I would count all of these things as 'progress'. But I am fine with whatever formulation you decide on in the end, some hint towards the previous grant is just useful because this is how we got the NFDI grant - based on the fact that it's complementing a bigger grant, this is pretty standard funding requirement ;-)